src/visibility-and-privacy.md

```diff
@@ -154,13 +154,18 @@ be a parent module of the item whose visibility is being declared.
 - `pub(self)` makes an item visible to the current module. This is equivalent
 to `pub(in self)`.
 
+> **Edition Differences**: Starting with the 2018 edition, paths for
+> `pub(in path)` must start with `crate`, `self`, or `super`.
```

Comment:
Isn't this just a consequence of the path having to be absolute?

Reply:
Yes, but there is nothing else that explicitly calls that out for visibility paths.  The only other part is the "use declarations", which only talk about the requirements in use declarations.  Or are you saying it should be worded differently?

Reply:
I didn't realise how the reference described paths in use items. The wording here is probably fine, but I think that there should be some wording on how paths that start with an identifier are resolved in the 2015 edition.

Reply:
Oops, good catch!